upstream/roles/detect_operators_in_cluster/tasks/op_ver.yml

```diff
@@ -61,7 +61,7 @@
     # curl -v -H "Accept: application/json" 'https://catalog.redhat.com/api/containers/v1/operators/indices?ocp_versions_range=v4.5&organization=community-operators'
     - name: "Pyxis"
       uri:
-        url: "https://catalog.redhat.com/api/containers/v1/operators/indices?ocp_versions_range={{ ocp_version_annotation_stdout }}&organization=community-operators"
+        url: "https://catalog.redhat.com/api/containers/v1/operators/indices?ocp_versions_range={{ ocp_version_annotation_stdout | urlencode }}&organization=community-operators"
```

Comment:
@Allda is this repo still used in community operators testing or it was already migrated code.
and if it is still used than is it master branch or different

Reply:
This code is still being used and we are going to migrate it soon. @ezopezo Do you know what branch is used by the community testing? Is it the master or a different one?